assets/js/nav.js

```diff
@@ -0,0 +1,32 @@
+const MOBILE_WIDTH_BREAKPOINT = 770;
+
+/* Hides or displays the nav menu items based on the size of the screen. */
+window.addEventListener('resize', function() {
+    const navItems = document.getElementById('menu-items');
+    if (window.innerWidth > MOBILE_WIDTH_BREAKPOINT) {
+        navItems.style.display = "block";
```

Comment:
Would it make sense to store `"block"` as a `const`?
Similarly, this can be done to `"none"`